src/python/WMCore/REST/Format.py

```diff
@@ -244,6 +244,7 @@ def stream_chunked(self, stream, etag, preamble, trailer):
                 trailer = None
                 raise
             finally:
+                print("ERROR, json.dumps failed to serialize %s, type %s" % (obj, type(obj)))
```

Comment:
I think Alan is right, doesn't finally block runs both exception and success case?